autogen.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Consider using `#!/bin/bash -e`. With it, autogen.sh will return non-zero status if something fails.